ghost/core/test/regression/models/model_posts.test.js

```
@@ -1611,10 +1611,12 @@ describe('Post Model', function () {
                 id: testUtils.DataGenerator.Content.users[0].id,
                 slug: testUtils.DataGenerator.Content.users[0].slug
             };
+            const staffCount = testUtils.DataGenerator.forKnex.users.length;
+            console.log('staffCount is', staffCount);
```

ESLint

[error] Unexpected console statement.

(no-console)